app.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+
+function Project(title, description, date, contributors, link){
```

Comment:
You won't get points deducted for this as this is technically correct, but you should change your constructor to be in line with starter code's constructor, i.e. here:
https://github.com/codefellows-seattle-301d18/03-jQuery-and-events/blob/master/starter-code/scripts/article.js

This will make it MUCH easier to instantiate your ``this`` properties in the future. I.e., see how you're instantiating it in Line 12? You don't need to do all that work with the examples in the starter code.